### PR TITLE
Add Sustain an Effect Exploration Activity

### LINKED ISSUE #12652: The "Sustain An Effect" Exploration Activity seems to be missing
While putting together a list of actions that I should put on a new player's character sheet I noticed that the exploration activity Sustain an Effect was not in the system yet. Below is the text so that someone can add it in.

SUSTAIN AN EFFECT

Tags: CONSENTRATE  EXPLORATION

You  Sustain  one  effect  with  a  sustained  duration  while 
moving at half speed. Most such effects can be sustained for 
10 minutes, though some specify they can be sustained for a 
different duration. Sustaining an effect that requires making 
complex decisions, such as  spectral weapon, can make you 
fatigued, as determined by the GM.

Source: Player Core pg. 439

### PR DESCRIPTION
**Pull request by DocSchlock**
Closes #12652

### CODE REVIEW

File: packs/actions/sustain-an-effect.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "system": {
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "interaction",
+        "description": {
+            "value": "<p>You Sustain one effect with a sustained duration while moving at half speed. Most such effects can be sustained for 10 minutes, though some specify they can be sustained for a different duration. Sustaining an effect that requires making complex decisions, such as spectral weapon, can make you @UUID[Compendium.pf2e.conditionitems.Item.Fatigued], as determined by the GM.</p>"
```
Comment (simonward) [inline packs/actions/sustain-an-effect.json:14]: Consider linking or emphasizing _spectral weapon_.

Comment (DocSchlock) [reply packs/actions/sustain-an-effect.json]: Added emphasis to spectral weapon.

**Comment by TikaelSol** [comment]
This was removed in the remaster

**Comment by TikaelSol** [comment]
Nevermind, the exploration activity is called that.

**Pull request merged**